pkg/Directory.Build.props

```diff
@@ -25,6 +25,7 @@
     <PackageProjectUrl>https://dot.net/ml</PackageProjectUrl>
     <PackageIconUrl>https://aka.ms/mlnetlogo</PackageIconUrl>
     <PackageReleaseNotes>https://aka.ms/mlnetreleasenotes</PackageReleaseNotes>
+    <PackageTags>ML.NET, ML, Machine Learning</PackageTags>
```

Comment:
Docs say space delimited actually - I'll find out whether there's a way to keep 'Machine Learning' as one tag.

Reply:
Yup, [space separated](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs#L760-L765)
I figure "Machine" and "Learning" are probably still useful separately.